### PR TITLE
add check if the mandatory command exist.

### PR DESCRIPTION
**Pull request by fujitatomoya**
**What type of PR is this?**

Cosmetic fix aligned with procedure, https://kubeedge.io/docs/advanced/debug/

/kind bug

**What this PR does / why we need it**:

It just adds the check if the mandatory command exists in the system.

**Which issue(s) this PR fixes**:

N.A

**Special notes for your reviewer**:

part of https://github.com/kubeedge/kubeedge/issues/5355

### CODE REVIEW

File: build/tools/certgen.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 set -o errexit
 
 readonly caPath=${CA_PATH:-/etc/kubeedge/ca}
-readonly caSubject=${CA_SUBJECT:-/C=CN/ST=Zhejiang/L=Hangzhou/O=KubeEdge/CN=kubeedge.io}
```
Comment (fujitatomoya) [inline build/tools/certgen.sh:6]: AFAIS, this is never used in anywhere else.

**Comment by fujitatomoya** [comment]
@chendave friendly ping.

**Comment by fujitatomoya** [comment]
@fisherxu @ZhengXinwei-F can you take a look at this? really minor fix.

File: build/tools/certgen.sh [hunk 2]
```diff
@@ -28,6 +27,15 @@ ensureFolder() {
     fi
 }
 
+ensureCommand() {
+    if command -v "$1" >/dev/null 2>&1; then
+        #echo "$1 exists."
```
Comment (fisherxu) [inline build/tools/certgen.sh:32]:
```suggestion
        echo "$1 exists."
```
This message should be necessary :)

File: build/tools/certgen.sh [hunk 3]
```diff
@@ -28,6 +27,15 @@ ensureFolder() {
     fi
 }
 
+ensureCommand() {
```
Comment (fisherxu) [inline build/tools/certgen.sh:30]:
```suggestion
ensureCommand() {
    echo "checking $1"
```

**Review by fisherxu — COMMENTED** [review]
Sorry for the delay, just some nits :) ptal @fujitatomoya

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**